### PR TITLE
Added eachmoodhtml

### PR DESCRIPTION
**Pull request by Swathik22**

### CODE REVIEW

File: scripts/moods.js [hunk 1]
```diff
@@ -38,4 +38,11 @@ getMoodHTML, taking mood
     -close div
     -return returnHTML
 
-*/
\ No newline at end of file
+*/
+const getMoodHTML=(mood)=>{
+    let moodHTML=`<div class="moodbox"></div>`
```
Comment (Lincoln-WIlcox) [inline scripts/moods.js]: class name should be cell, and put content within div

**Review by Lincoln-WIlcox — CHANGES_REQUESTED** [review]

File: scripts/moods.js [hunk 2]
```diff
@@ -38,4 +38,11 @@ getMoodHTML, taking mood
...
+    moodHTML+=`<div class="moodImg">${mood.imageUrl}</div>
+            <div class="moodName">${mood.name}</div>
```
Comment (joshuahamrick) [inline scripts/moods.js]: Please put mood name before the mood image!

File: scripts/moods.js [hunk 3]
```diff
@@ -38,4 +38,12 @@ getMoodHTML, taking mood
...
     -return returnHTML
 
-*/
\ No newline at end of file
+*/
+const getMoodHTML=(mood)=>{
+    let moodHTML=`<div class="cell">`
+    moodHTML+=`<div class="moodImg">${mood.imageUrl}</div>
+            <div class="moodName">${mood.name}</div>
+            <div class="moodQuotes">${mood.quotes[0]}\n ${mood.quotes[1]}</div>
```
Comment (joshuahamrick) [inline scripts/moods.js]: please make a div to contain this div(#moodQuotes) and also a paragraph with the string "Macho Advice:"

**Review by Lincoln-WIlcox — APPROVED** [review]

**Review by joshuahamrick — APPROVED** [review]
Beautiful

**Pull request merged**